### ISSUE TITLE
Use a Apple Developer ID to sign the Mac App

### ISSUE THREAD

**Issue by readefries** [bunqCommunity/bunqDesktop#337]
Currently, the Mac App is unsigned. As a result, people have to perform extra steps to install the App on a Mac.

I already saw that [signing of Electron Apps](https://pracucci.com/atom-electron-signing-mac-app.html) is possible, this shouldn't be hard to implement in Travis.

**Comment by Crecket** [comment]
It's more about me not being a Mac user and not having access to a developer account right now. But as I stated in #293 I wan't to have all the releases code signed before the final 1.0.0 release so it is on the roadmap :+1:

**Comment by readefries** [comment]
I have both, but I think we should create a specific 'Developer' for the project, and not bind it to a person.

**Comment by Crecket** [comment]
I agree, I'll be looking into what would be the best way to get code-signing certificates for the other platforms. If I remember correctly Apple only allows their own certificates so we might be able to re-use those for the linxu/windows releases.

Haven't done a lot of research into their procedures and limitations though so I'll have to look into it
